cg/store/models.py

```diff
@@ -969,7 +969,7 @@ class RunDevice(Base):
     type: Mapped[DeviceType]
     internal_id: Mapped[UniqueStr64]
 
-    run_metrics: Mapped[list["RunMetrics"]] = orm.relationship(
+    instrument_run: Mapped[list["InstrumentRun"]] = orm.relationship(
```

Comment:
Plural here since it returns a list
```suggestion
    instrument_runs: Mapped[list["InstrumentRun"]] = orm.relationship(
```